package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "jhipster-registry",
-  "version": "7.0.0",
+  "version": "7.2.0",
```

Comment:
@pascalgrimaud , how do you apply next version? `npm version` or something else?

Reply:
When I want to release:
- I edit package.json by upgrading the version
- I launch npm install, so the package lock is upgraded too

Reply:
@pascalgrimaud thanks for confirmation.

@tarkil, can you please change version to `7.1.0` across all places

Reply:
```suggestion
  "version": "7.1.0",
```

Reply:
should be unchanged (so 7.0.0), as this version will be updated when I'll do the release

Reply:
In this PR @tarkil  is updating version in advance at all places. Ideally, I don't like to do this, but, it doesn't interfere with @pascalgrimaud  release flow as we just need to execute `npm install` and apply tag. I will leave final decision with @pascalgrimaud  to continue with suggestions or revert at all places

Reply:
IMHO, the decision to update the version is yours. So, I have reverted the version to 7.0.0.